note-app/src/Component/Main/Main.jsx

```diff
@@ -92,7 +104,22 @@ const Main = () => {
           }
          </div>
        )}
-       <div className='editorIcon-btn f-m ml-s'><MdOutlineNewLabel /></div>
+       <div className='editorIcon-btn f-m ml-s' onClick={() => dispatch({type: "TAGPALLET"})}><MdOutlineNewLabel /></div>
+       {tagPalletVisible && (
+         <div className='tagPallet-container flex'>
+         {
+           tagPallets.map(({id,tag}) =>(
+            <div key={id}
+              className='tagCont f-s font-l'
+               onClick={() => dispatch({type: "SET_TAG",payload: tag})}
+            >
+              {tag}
+            </div>
+           ))
+         }
+         </div>
+       )}
+         {tags !== '' && <div className='displayTags mt-s ml-s f-s font-l p-xs'>{tags}</div>}
```

Comment:
You can do directly `{tags && <div></div>}`

Reply:
it show the background of tags in the ui if it is empty. so i use that way